latest/index.bs

```diff
@@ -175,7 +175,174 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
+        |       |
+        |       ├── data      # You MUST add a one-dimensional zarr array named "data". 
+        |       |   |         # `data` MAY be chunked as the user desires.
+        |       |   |
+        |       |   ├── 0
+        │       │   |   ...
+        │       |   └── n
+        |       |
+        |       ├── indices   # You MUST add a one-dimensional zarr array named "indices".
+        |       |   |         # `indices` MAY be chunked as the user desires.
+        |       |   |
+        |       |   ├── 0
+        │       │   |   ...
+        │       |   └── n
+        |       |
+        |       └── indptr    # You MUST add a one-dimensional zarr array named "indptr".
+        |           |         # `indptr` MAY be chunked as the user desires.
+        |           |
+        |           ├── 0
+        │           |   ...
+        │           └── n
+        |
+        ├── varp              # You MAY add a varp group comtainer. The varp group contains sparse arrays that annotate the columns in X.
+        |   │                 # The rows in each array MUST be index-matched to the columns in X (if present). 
+        |   |
+        │   ├── .zgroup
+        |   |
+        |   ├── .zattrs       # `.zattrs` MAY contain `"encoding-type"`, which is set to `"dict"` by AnnData.           
+        |   │                 # `.zattrs` MAY contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
+        |   │
+        │   └── varp_0        # You MAY add a zarr group for each varp array.
+        |       |             # Each varp array MUST have the same number of rows as columns in X.
+        |       |
+        │       ├── .zgroup
+        |       |
+        |       ├── .zattrs   # `.zattrs` MAY contain `"encoding-type"`, which is set to `"csr_matrix"` by AnnData.           
+        |       │             # `.zattrs` MAY contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
+        |       |             # `.zattrs` MAY contain `"shape"` which is an array giving the shape of the densified array.
+        |       |
+        |       ├── data      # You MUST add a one-dimensional zarr array named "data". 
+        |       |   |         # `data` MAY be chunked as the user desires.
+        |       |   |
+        |       |   ├── 0
+        │       │   |   ...
+        │       |   └── n
+        |       |
+        |       ├── indices   # You MUST add a one-dimensional zarr array named "indices".
+        |       |   |         # `indices` MAY be chunked as the user desires.
+        |       |   |
+        |       |   ├── 0
+        │       │   |   ...
+        │       |   └── n
+        |       |
+        |       └── indptr    # You MUST add a one-dimensional zarr array named "indptr".
+        |           |         # `indptr` MAY be chunked as the user desires.
+        |           |
+        |           ├── 0
+        │           |   ...
+        │           └── n
+        └── uns
```

Comment:
@giovp and @ivirshup, do you think `uns`should be included? The examples in [this comment](https://github.com/ome/ngff/issues/99#issue-1127549046), it looks like all of those things can be stored elsewhere (e.g., store the image in an image group and that would contain the associated metadata). However, when I look at the results of the [squidpy visum tutorial](﻿﻿https://squidpy.readthedocs.io/en/stable/auto_tutorials/tutorial_visium_fluo.html), looks like there are a lot of other things stored in there (e.g., analysis parameters).

If we include it, should we basically say it:
- MAY contain groups, which may contain groups, sparse arrays, and dense arrays
- MAY contain sparse arrays, and dense arrays

Reply:
yes indeed, it's essentially a containers for metadata during analysis (e.g. color palette, method parameters, but even analysis results that don't fit in obsm/varm etc.). I don't think we ever store sparse arrays there, for sure dense arrays, but also dataframes, dictionaries with various k,v types.

Reply:
We would also favor images in Zarr groups rather than `uns` (or only a reference link in `uns`). We had already all microscopy in NGFF Zarr before adding AnnData on top for annotations. Embedding another copy of all imagery inside of AnnData felt not so practical for our use case.

Reply:
> We would also favor images in Zarr groups rather than uns (or only a reference link in uns). We had already all microscopy in NGFF Zarr before adding AnnData on top for annotations. Embedding another copy of all imagery inside of AnnData felt not so practical for our use case.

💯 agree we should move out all images (and related metadata) from `adata.uns` to zarr groups.

Reply:
> We would also favor images in Zarr groups rather than uns (or only a reference link in uns).

Totally agree. As @giovp said, we should be able to completely move away from storing images in `uns` (along with scale, transform, etc. data that is associated with said images).

See [this comment](https://github.com/ome/ngff/pull/64/files#r901580502) for how we think a table can be linked to an image in another zarr group. Does that seem reasonable?

Reply:
> yes indeed, it's essentially a containers for metadata during analysis (e.g. color palette, method parameters, but even analysis results that don't fit in obsm/varm etc.). I don't think we ever store sparse arrays there, for sure dense arrays, but also dataframes, dictionaries with various k,v types.

Makes sense, @giovp . I've updated the spec to reflect this. I've included sparse arrays since from a brief look at the anndata code it looks like it would work. Does this seem right?

Reply:
> do you think uns should be included?

I'm not sure I see a way around it. I think many of the things we currently store there will eventually move out into better representations (e.g. metadata on individual arrays) but it's an important catch all currently.

> I don't think we ever store sparse arrays there,

We stored sparse adjacency matrices there before we added `obsp`.

Reply:
Makes sense to me. +1 for moving things when we can, but I understand that there will likely always be "miscellaneous" things.